dffml/service/dev.py

```diff
@@ -1039,6 +1040,135 @@ async def run(self):
                 )
 
 
+class CommitLintError(Exception):
+    pass
+
+
+class CommitLint(CMD):
+    """
+    Enforce commit message style 
+    """
+
+    def _get_ignore_filter(self):
+        return lambda x: not x.endswith("_")
+
+    def _execute_func_list(self, param, func_list):
+        for func in func_list:
+            param = func(param)
+        return param
+
+    async def _get_file_mutations(self):
+        no_mutation = lambda x: x
+        mutation_func_factory = lambda ext: lambda x: x + ext
+        extensions = await self._get_all_exts()
+        mutations = {
+            "prefix_mutations": [
+                no_mutation,
+                lambda x: "." + x,
+                lambda x: "dffml/" + x,
+            ],
+            "suffix_mutations": [
+                no_mutation,
+                *[mutation_func_factory(ext) for ext in extensions],
+            ],
+        }
+        return mutations
+
+    async def _get_cmd_output(self, cmd: List[str]):
```

Comment:
We might want to put this in `dffml/util/`, since this is a common pattern

Reply:
Probably in `dffm/util/os.py`, or a more specifically named file

Reply:
Made an issue for that #1078